cli/install/install.go

```diff
@@ -472,7 +533,6 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 		return err
 	}
 	os.Chmod(path, defaultDirPermissions)
-
```

Comment:
This change is not related to the feature implementation.

Reply:
fixed